ticdc/manage-ticdc.md

```diff
@@ -220,6 +220,24 @@ The following are descriptions of parameters that can be configured for the sink
 
 For more parameters of Pulsar, see [pulsar-client-go ClientOptions](https://godoc.org/github.com/apache/pulsar-client-go/pulsar#ClientOptions) and [pulsar-client-go ProducerOptions](https://godoc.org/github.com/apache/pulsar-client-go/pulsar#ProducerOptions).
 
+#### Configure sink URI with cdclog
+
+The cdclog files can be used together with `br` to provide point-in-time (PiTR) recovery. This is experimental feature.
```

Comment:
@amyangfei Here "cdclog" is mentioned for the first time in the entire docs repo. I think we need to explain what "cdclog" is, what it is used for, etc.